app/views/worklogs/_below_worklog_index.html.slim

```diff
@@ -13,7 +13,7 @@
         p.lead.pull-right Client
       .col-md-10
         select.col-md-4.client-filter.form-control
-          option data-url="#{worklogs_path(merge_params(client: nil))}"
+          option data-url="#{worklogs_path(merge_params(client: nil))}" value="" ng-if="false"
```

Comment:
Ahh, now I see what you mean.
The code works.
By adding ng-if="false" we disable the empty-option forever AND make angularjs think a change happened. We pretty much trigger the method to update the entries :)